ClientRuntimes/CSharp/ClientRuntime.Azure/Microsoft.Rest.ClientRuntime.Azure/project.lock.json

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
Is this file necessary, or is it created by the build process?  I assume this is providing the dependency versions for all dependent assemblies